modules/10-basics/50-syntax-errors/description.en.yml

```diff
@@ -0,0 +1,28 @@
+---
+
+name: Syntax errors
+theory: |
+  If a PHP program isn't syntactically correct, interpreter will show a relevant message and note with a file and line where an error might occur. **Syntax errors** occur when the code has grammar mistakes. Unlike human languages where grammar rules are important and the lack of it won't affect your ability to read or understand any text; everything in programming is strict. Even a tiny violation and your program will not run. Some examples could be, a wrong parenthesis order, a `;` that you forgot to add or other details.
```

Comment:
- "a message [...] **a** note" (without an article note seems like a verb here)
- "an error might occur" — wrong tense, the error already occurred. "Might have occurred" would suit better.
- "and the lack of" — should be "but", not "and"
- "...text; everything..." — I don't think the semicolon is needed here, it splits two parts of the same sentence, which is confusing (Unlike..., bla)
- "Some examples could be" — in cases like this feel free to rephrase the sentence to make it more natural in the English context. No need to preserve the Russian constructs. Also, ditch the comma :)

Reply:
* "where an error might occur" - probably it's better to remove this 'might' word and just say "where an error occured" 
* "and the lack of" - this sentence sounds weird with both "but" and "and", I'll just rephrase it completely.
* "Some examples could be" - yeah, I'll rephrase this one too. btw those commas were suggested by a native speaker :)